Index: git/src/agent.c
===================================================================
--- git.orig/src/agent.c	2012-09-08 12:09:29.000000000 +0100
+++ git/src/agent.c	2012-09-11 08:55:18.875632643 +0100
@@ -428,7 +428,7 @@
 	}
 
 	if (dbus_connection_send_with_reply(connection, message,
-						&call, -1) == FALSE) {
+						&call, 3*60*1000) == FALSE) {
 		dbus_message_unref(message);
 		g_free(passphrase_reply);
 		return -ESRCH;
