Index: git/configure.ac
===================================================================
--- git.orig/configure.ac	2012-05-25 09:00:25.279958309 +0100
+++ git/configure.ac	2012-05-25 09:00:25.643956513 +0100
@@ -33,8 +33,8 @@
                              gupnp-1.0           >= $GUPNP_REQUIRED
                              gupnp-av-1.0        >= $GUPNP_AV_REQUIRED
                              gupnp-vala-1.0      >= $GUPNP_VALA_REQUIRED
-                             tracker-miner-0.10
-                             tracker-sparql-0.10"
+                             tracker-miner-0.14
+                             tracker-sparql-0.14"
 
 PKG_CHECK_MODULES(TRACKER_MINER_UPNP, [$TRACKER_MINER_UPNP_REQUIRED])
 
Index: git/src/Makefile.am
===================================================================
--- git.orig/src/Makefile.am	2012-05-25 09:00:25.000000000 +0100
+++ git/src/Makefile.am	2012-05-25 09:04:58.230597213 +0100
@@ -7,8 +7,8 @@
 	--pkg gio-2.0                                  \
 	--pkg gupnp-1.0                                \
 	--pkg gupnp-av-1.0                             \
-	--pkg tracker-sparql-0.10                      \
-	--pkg tracker-miner-0.10                       \
+	--pkg tracker-sparql-0.14                      \
+	--pkg tracker-miner-0.14                       \
 	$(BUILD_VALAFLAGS)
 
 AM_CPPFLAGS =                                          \
