Index: lirc-0.9.0/drivers/lirc_serial/lirc_serial.c
===================================================================
--- lirc-0.9.0.orig/drivers/lirc_serial/lirc_serial.c	2011-03-25 22:28:18.000000000 +0000
+++ lirc-0.9.0/drivers/lirc_serial/lirc_serial.c	2012-10-11 22:34:54.842803107 +0100
@@ -66,7 +66,6 @@
 #include <linux/poll.h>
 #include <linux/platform_device.h>
 
-#include <asm/system.h>
 #include <linux/io.h>
 #include <linux/irq.h>
 #include <linux/fcntl.h>
