Index: git/cogl/cogl.c
===================================================================
--- git.orig/cogl/cogl.c	2012-09-07 14:27:53.000000000 +0100
+++ git/cogl/cogl.c	2012-09-07 15:22:24.819321776 +0100
@@ -25,6 +25,10 @@
 #include "config.h"
 #endif
 
+#ifdef HAVE_BCM_HOST
+#include <bcm_host.h>
+#endif
+
 #include <string.h>
 #include <math.h>
 #include <stdlib.h>
@@ -796,6 +800,10 @@
 
   if (g_once_init_enter (&init_status))
     {
+#ifdef HAVE_BCM_HOST
+      bcm_host_init();
+#endif
+
       bindtextdomain (GETTEXT_PACKAGE, COGL_LOCALEDIR);
       bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 
Index: git/configure.ac
===================================================================
--- git.orig/configure.ac	2012-09-07 14:56:58.000000000 +0100
+++ git/configure.ac	2012-09-07 15:58:22.322652540 +0100
@@ -850,7 +850,19 @@
 AS_IF([test "x$NEED_EGL" = "xyes" && test "x$EGL_CHECKED" != "xyes"],
       [
         PKG_CHECK_EXISTS([egl],
-          [COGL_PKG_REQUIRES="$COGL_PKG_REQUIRES egl"],
+          [
+	    COGL_PKG_REQUIRES="$COGL_PKG_REQUIRES egl"
+	    SAVED_CFLAGS="$CFLAGS"
+	    PKG_CHECK_MODULES(EGL, egl)
+	    CFLAGS="$CFLAGS $EGL_CFLAGS"
+	    AC_CHECK_HEADER([bcm_host.h], [platform_bcm=yes], [platform_bcm=no])
+	      AS_IF([test "x$platform_bcm" = "xyes"],
+                [
+                  COGL_EXTRA_LDFLAGS="$COGL_EXTRA_LDFLAGS -lbcm_host"
+		  AC_DEFINE([HAVE_BCM_HOST], [1], [Whether we have libbcm_host])
+                ])
+	    CFLAGS="$SAVED_CFLAGS"
+	  ],
           [
             AC_CHECK_HEADERS(
               [EGL/egl.h],
